Wrapping/Generators/SwigInterface/igenerator.py

```diff
@@ -2024,14 +2039,23 @@ def generate_swig_input(submoduleName, classes):
             outputPYIHeaderFile, outputPYIMethodFile, classes[itk_class]
         )
 
-        write_class_pyi(
-            Path(f"{options.pyi_dir}/{classes[itk_class].submodule_name}.pyi"),
+        # write_class_pyi(
+        #     Path(f"{options.pyi_dir}/{classes[itk_class].submodule_name}.pyi"),
+        #     itk_class,
+        #     outputPYIHeaderFile.getvalue(),
+        #     outputPYIMethodFile.getvalue(),
+        # )
```

Comment:
Remove commented code

Reply:
Sorry! Just noticed that as well. The commits should be fixed.